codespell_lib/data/dictionary.txt

```diff
@@ -22233,6 +22275,8 @@ neibors->neighbors
 neice->niece, nice,
 neigbhorhood->neighborhood
 neigbhorhoods->neighborhoods
+neigbhour->neighbor
+neigbhours->neighbors
```

Comment:
I think you should suggest `en-GB` and `en-US` separately. Try not to mix both `bour` and `bor`
```
neigbhour->neighbour
neigbhours->neighbours
neigbhor->neighbor
neigbhors->neighbors
```
* https://github.com/search?q=neigbhour&type=code  (3K)
* https://github.com/search?q=neigbhours&type=code  (1K)
* https://github.com/search?q=neigbhor&type=code  (3K)
* https://github.com/search?q=neigbhors&type=code  (7K)